assets/js/components/api-error-placeholder/error-message.js

```diff
@@ -5,11 +5,21 @@ import { __ } from '@wordpress/i18n';
 import PropTypes from 'prop-types';
 import { escapeHTML } from '@wordpress/escape-html';
 
-const getErrorMessage = ( { apiMessage, message } ) => {
+const getErrorMessage = ( { apiMessage, message, frontendMessage } ) => {
```

Comment:
Similar comment here as I left elsewhere.  I think `frontendMessage` is _too_ specific a variable name.  I anticipate this may be a type of message left in non frontend contexts as well.  Would `userFriendlyMessage` or `informalMessage` be a better fit here?